radixdlt-java/src/main/java/com/radixdlt/client/core/RadixUniverse.java

```diff
@@ -66,28 +78,43 @@
 	 * Initializes the default universe with a Peer Discovery mechanism.
```

Comment:
"with a Peer Discovery mechanism" should be changed to "from a set of seed nodes" or similar

Reply:
Keyword being *can*.
Not very important anyway, but I do think it should be "from a set of seed nodes", since the seeds don't necessarily have to come from a peer discovery mechanism.

Reply:
fair enough